pkg/web/static/js/transactionAcceptPreview.js

```diff
@@ -75,4 +75,174 @@ function setIdentifierData(el, identifier) {
+          break;
+        // Set the beneficiary address line.
+        case key.startsWith('address_bf_'):
+          beneficiaryPerson.geographic_addresses[0].address_line.push(params[key]);
+          break;
+        // Set the beneficiary country and address type.
+        case key.startsWith('addr_bf_'):
+          beneficiaryPerson.geographic_addresses[0][newKey] = params[key];
+          break;
+        // Set details for the beneficiary natural person that's not a name identifier or geographic address.
+        case key.startsWith('np_bf_'):
+          beneficiaryPerson[newKey] = params[key];
+          break;
+        // Set the beneficiary account number.
+        case key.startsWith('acct_bf_'):
+          beneficiaryPerson.account_numbers.push(params[key]);
+          break;
+        // Set the originating VASP name identifiers and name identifier type.
+        case key.startsWith('id_orig'):
+          originatingVASP.name.name_identifiers[0][newKey] = params[key];
+          break;
+        // Set the originating VASP address line.
+        case key.startsWith('address_orig'):
+          originatingVASP.geographic_addresses[0].address_line.push(params[key]);
+          break;
+        // Set the originating VASP country and address type.
+        case key.startsWith('addr_orig'):
+          originatingVASP.geographic_addresses[0][newKey] = params[key];
+          break;
+        // Set the originating VASP national identification.
+        case key.startsWith('nat_orig'):
+          originatingVASP.national_identification[newKey] = params[key];
+          break;
+        // Set the originating VASP country of registration.
+        case key.startsWith('ctry_orig'):
+          originatingVASP[newKey] = params[key];
+          break;
+        // Set the beneficiary VASP name identifiers and name identifier type.
+        case key.startsWith('id_benf'):
+          beneficiaryVASP.name.name_identifiers[0][newKey] = params[key];
+          break;
+        // Set the beneficiary VASP address line.
+        case key.startsWith('address_benf'):
+          beneficiaryVASP.geographic_addresses[0].address_line.push(params[key]);
+          break;
+        // Set the beneficiary VASP country and address type.
+        case key.startsWith('addr_benf'):
+          beneficiaryVASP.geographic_addresses[0][newKey] = params[key];
+          break;
+        // Set the beneficiary VASP national identification.
+        case key.startsWith('nat_benf'):
+          beneficiaryVASP.national_identification[newKey] = params[key];
+          break;
+        // Set the beneficiary VASP country of registration.
+        case key.startsWith('ctry_benf'):
+          beneficiaryVASP[newKey] = params[key];
+          break;
+      };
+    };
+
+    // Convert transaction amount to a float. If conversion fails, set amount to the original value.
+    const amount = parseFloat(data.transaction.amount);
```

Comment:
Is the `parseFloat` function unique to this codebase or a JS thing? I'm wondering under what conditions the type conversion would fail, and is that possible to validate before trying to convert it?. Possibly this is just me not understanding the context for which this PR fits into the larger picture.

Reply:
`parseFloat` is a JS function that will return the floated value of a string or NaN if the string can't be converted to a number. Here's a [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) for more more info. 

When submitting the form, the `amount` must be sent as a float or a parse error will be returned from the BE.

Reply:
Ok, is it possible to return an error to the user when the form is submitted instead of silently failing?

Reply:
Gotcha, we should try to catch that in the frontend if possible to avoid unnecessary requests to the backend. If we don't have that kind of validation yet on the frontend I understand, but let's create a story for that at least if it's too much for this PR.

Reply:
I'll create a story so that this may be included when validation gets added to the other forms.